### PR TITLE
Tweak to battle royale crate drop rates

### PR DESCRIPTION
**Pull request by Moltijoe**
Battle royale, but you can actually walk around

:cl:  
tweak: Battle Royale
/:cl:

### CODE REVIEW

**Comment by Yarinoi** [comment]
is this the pr were supposed to beg to be merged

**Comment by Moltijoe** [comment]
> is this the pr were supposed to beg to be merged

yes, but i'm making another tweak real quick

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm [hunk 1]
```diff
@@ -373,11 +379,11 @@ GLOBAL_VAR(final_zone)
 	name = "very cool tie (do not remove)"
 	desc = "Totally not just here for keeping track of kills."
 	var/datum/antagonist/battleroyale/last_hit
-	clothing_traits = (TRAIT_NODROP)
 	resistance_flags = INDESTRUCTIBLE //no escaping
 
 /obj/item/clothing/neck/tie/gamer/equipped(mob/user, slot)
 	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, "I will kill you if you take this off somehow") //can't be a clothing trait because those get applied to the mob, not the item
```
Comment (iamthesilly) [inline yogstation/code/game/gamemodes/battle_royale/battleroyale.dm:386]: Shouldn't this be in initialize?

Comment (Moltijoe) [reply yogstation/code/game/gamemodes/battle_royale/battleroyale.dm:386]: ... probably? i can't remember why i made it equipped in the first place

**Review by SapphicOverload — APPROVED** [review]
i forgor

**Pull request merged**